contracts/providers/chainlink/ChainlinkPairAggregator.sol

```diff
@@ -19,15 +19,32 @@ import "@chainlink/contracts/src/v0.5/interfaces/AggregatorInterface.sol";
 import "../../interfaces/PairAggregatorInterface.sol";
 
 contract ChainlinkPairAggregator is PairAggregatorInterface {
+
+    uint256 internal constant TEN = 10;
+
     AggregatorInterface public aggregator;
+    uint8 public tokenDecimals;
+    uint8 public responseDecimals;
 
-    constructor(address aggregatorAddress) public {
-        require(aggregatorAddress != address(0x0), "Aggregator address is required.");
+    constructor(address aggregatorAddress, uint8 tokenDecimalsValue, uint8 responseDecimalsValue) public {
+        require(aggregatorAddress != address(0x0), "PROVIDE_AGGREGATOR_ADDRESS");
+        require(tokenDecimalsValue > 0, "PROVIDE_VALID_TOKEN_DECIMALS");
+        require(responseDecimalsValue > 0, "PROVIDE_VALID_RESPONSE_DECIMALS");
```

Comment:
Is it guaranteed that chainlink never has a 0 value here?

Reply:
They have mentioned today, the response is similar to token decimals. So it might be zero.
I removed the validation.